java/algorithms/BellmanFord-FunSat01/Main.java

```diff
@@ -0,0 +1,109 @@
+import org.sosy_lab.sv_benchmarks.Verifier;
+
```

Comment:
General comment: I think all files need a clear comment under which license the modifications are published.
Further, there should be a clear indication whether the files is modified or not and how is the author of the modifications.